modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -235,12 +235,12 @@ def validate_service_dates
       end
     end
 
+    # do not clear out 'auth_headers' or 'file_data' attributes
+    # 'auth_headers' is required to make calls to EVSS for uploading docs
+    # 'file_data' is required to know what doc to upload to EVSS
+    # See API-14303
     def remove_encrypted_fields
```

Comment:
just to add more explicitness to the `after_validation` statement [this is called from](https://github.com/department-of-veterans-affairs/vets-api/pull/9466/files#diff-cecc476b1562949c65821a873933ee4c2cf286b5023f0e6b06f97c2b3fc5a3cbL22)

Reply:
I really do like that idea.  I think in the interest of getting a fix out there ASAP, we should maybe handle that in another upcoming ticket/PR.  If that's cool?

Reply:
Sounds good to me!